example/src/index.js

```diff
@@ -86,15 +109,26 @@ class Example extends React.Component {
           onChange={this.handleChange}
           onClickLink={href => console.log("Clicked link: ", href)}
           onClickHashtag={tag => console.log("Clicked hashtag: ", tag)}
+          onCreateLink={title => {
+            // Delay to simulate time taken to upload
+            return new Promise((resolve, reject) => {
+              setTimeout(() => {
+                if (title !== "error") {
+                  return resolve(
+                    `https://dummyimage.com/600x400/000/fff&text=${title}`
```

Comment:
Should probably update to something more realistic

```suggestion
                    `/doc/${encodeURIComponent(title)}`
```

Reply:
It would just provide a "realistic" looking document link:

![image](https://user-images.githubusercontent.com/380914/84340803-60ee7980-ab56-11ea-94b6-525194c1f463.png)

I'm saying don't use an image url for something that's supposed to be a doc url.